ci/build/build-docker-images.py

```diff
@@ -43,8 +44,9 @@
     "cu112": "nvidia/cuda:11.2.0-cudnn8-devel-ubuntu20.04",
     "cu111": "nvidia/cuda:11.1.1-cudnn8-devel-ubuntu20.04",
     "cu110": "nvidia/cuda:11.0.3-cudnn8-devel-ubuntu20.04",
-    "cu102": "nvidia/cuda:10.2-cudnn8-devel-ubuntu20.04",
```

Comment:
Nice catch. I think this is orthogonal to the py3.11 change. Let's make a PR to fix this first?

Reply:
https://github.com/ray-project/ray/pull/31385